templates/pitch-donors.html

```diff
@@ -1,8 +1,8 @@
 <p>{{ _(
     "People who contribute to the commons need you to support their work. "
-    "Building free software, spreading free knowledge, these things take "
-    "time and cost money, not only to do the initial work, but also to "
-    "maintain it over time."
+    "Building libre software, spreading free knowledge, these things take "
```

Comment:
To make the point of not being costless. "free" ambiguity gets even worse when translated specifically as "gratis" in other languages, and it does.